### ISSUE TITLE
Meadowphysics has reference to screen.c, breaks compilation

### ISSUE THREAD

**Issue by ghost** [scanner-darkly/multipass#3]
The meadowphysics config.mk has a reference to screen.c which I don't think is supposed to be in there :) Removing it enabled me to compile.

Cheers,
Forrest

**Comment by ghost** [comment]
Considering making a fork to fix this and add the missing flash.sh files from the official monome module repos, would that be helpful?

Seems like the only way to put in a PR.

**Comment by scanner-darkly** [comment]
thank you - this is super helpful. yeah, you'll probably have to fork first before making a PR unless i add you as a collaborator. i can add flash.sh if that's the only change.

re: original issue - could you post the error message you were getting? 

it shouldn't execute any screen functions as per config https://github.com/scanner-darkly/multipass/blob/master/monome_euro/meadowphysics/module.h#L55 so i included it to avoid build errors - but perhaps the compiler optimizes those away anyway?

**Comment by ghost** [comment]
Here is what I get:

```
./libavr32/src/screen.c: In function 'write_command':
../libavr32/src/screen.c:45: error: 'OLED_SPI' undeclared (first use in this function)
../libavr32/src/screen.c:45: note: each undeclared identifier is reported only once for each function it appears in
../libavr32/src/screen.c:45: error: 'OLED_SPI_NPCS' undeclared (first use in this function)
../libavr32/src/screen.c:47: error: 'OLED_DC_PIN' undeclared (first use in this function)
../libavr32/src/screen.c: In function 'write_data':
../libavr32/src/screen.c:56: error: 'OLED_SPI' undeclared (first use in this function)
../libavr32/src/screen.c:56: error: 'OLED_SPI_NPCS' undeclared (first use in this function)
../libavr32/src/screen.c:58: error: 'OLED_DC_PIN' undeclared (first use in this function)
../libavr32/src/screen.c: In function 'screen_set_rect_2':
../libavr32/src/screen.c:91: error: 'OLED_DC_PIN' undeclared (first use in this function)
../libavr32/src/screen.c:92: error: 'OLED_SPI' undeclared (first use in this function)
../libavr32/src/screen.c:92: error: 'OLED_SPI_NPCS' undeclared (first use in this function)
../libavr32/src/screen.c: In function 'writeScreenBuffer1':
../libavr32/src/screen.c:130: error: 'OLED_SPI' undeclared (first use in this function)
../libavr32/src/screen.c:130: error: 'OLED_SPI_NPCS' undeclared (first use in this function)
../libavr32/src/screen.c:132: error: 'OLED_DC_PIN' undeclared (first use in this function)
../libavr32/src/screen.c: In function 'writeScreenBuffer2':
../libavr32/src/screen.c:149: error: 'OLED_DC_PIN' undeclared (first use in this function)
../libavr32/src/screen.c:150: error: 'OLED_SPI' undeclared (first use in this function)
../libavr32/src/screen.c:150: error: 'OLED_SPI_NPCS' undeclared (first use in this function)
../libavr32/src/screen.c: In function 'screen_clear1':
../libavr32/src/screen.c:179: error: 'OLED_SPI' undeclared (first use in this function)
../libavr32/src/screen.c:179: error: 'OLED_SPI_NPCS' undeclared (first use in this function)
../libavr32/src/screen.c:181: error: 'OLED_DC_PIN' undeclared (first use in this function)
../libavr32/src/screen.c: In function 'screen_clear2':
../libavr32/src/screen.c:193: error: 'OLED_DC_PIN' undeclared (first use in this function)
../libavr32/src/screen.c:194: error: 'OLED_SPI' undeclared (first use in this function)
../libavr32/src/screen.c:194: error: 'OLED_SPI_NPCS' undeclared (first use in this function)
../libavr32/src/screen.c: In function 'init_oled':
../libavr32/src/screen.c:235: error: 'OLED_RES_PIN' undeclared (first use in this function)
make: *** [../libavr32/src/screen.o] Error 1
```

I figure the flash.sh files are module-dependent, so there would be 5 (one for each module) to add.. I'll go ahead and do this change in a fork and submit it that way.

**Comment by scanner-darkly** [comment]
yeah weird that i only had this in MP but not ES / WW configs. thanks for the PR, merged!

**Comment by scanner-darkly** [comment]
fixed in 540984bf0a1481ba7f7158b16b0b550798357aa9
